fleeto_backend/app/controllers/customer/v1/locations_controller.rb

```diff
@@ -5,11 +5,47 @@ def set_location
     head :no_content
   end
 
+  def distance_matrix_to_drop_off_location
+    customer_location = Customers::LocationService.new(current_customer).get_location
+    service = Map::Factory.new_service_adaptor(:distance_matrix).new(from: customer_location, to: {long: params[:long], lat: params[:lat]})
```

Comment:
Too long line
Maybe initialize the @service in a before action since it's used in many actions?